flight_monitor.py

```
@@ -0,0 +1,800 @@
+    def __init__(self, config_file: str = "flight_config.json"):
+        self.config_file = config_file
+        self.config = self.load_config()
+        self.setup_logging()
+        
+    def load_config(self) -> Dict:
+        """Load configuration from JSON file"""
+        try:
+            with open(self.config_file, 'r') as f:
+                return json.load(f)
+        except FileNotFoundError:
+            raise FileNotFoundError(f"Configuration file {self.config_file} not found")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)